templates/zerver/emails/confirm_registration_mit.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
```

Comment:
@RameshAditya  sorry, disregard these comments, since the fix is outside of the scope of this PR. I will fix them later or if you want to tackle them as well.